src/Generator/ClassGenerator.php

```diff
@@ -283,7 +280,7 @@ public function getName()
     }
 
     /**
-     * @param  string $namespaceName
+     * @param  ?string $namespaceName
```

Comment:
There is an existing call in the code: `setNamespaceName(null)`, so param should be nullable.